microlith/entrypoints/webserver.sh

```diff
@@ -14,4 +14,10 @@
 # limitations under the License.
 
 set -ex
```

Comment:
Still want `-x` set?

Reply:
webserver is simple enough that I think it's useful